crates/kitsune_p2p/kitsune_p2p/src/spawn/actor/space.rs

```diff
@@ -1249,7 +1261,12 @@ impl KitsuneP2pHandler for Space {
         _space: Arc<KitsuneSpace>,
         basis: Arc<KitsuneBasis>,
     ) -> KitsuneP2pHandlerResult<bool> {
+        tracing::info!("Have local joined agents {:?}", self.local_joined_agents);
         let loc = basis.get_loc();
+        tracing::info!(
+            "Doing basis check against num arcs {}",
+            self.agent_arqs.len()
+        );
```

Comment:
Feels debuggy to me but I won't argue it

```suggestion
        tracing::debug!("Have local joined agents {:?}", self.local_joined_agents);
        let loc = basis.get_loc();
        tracing::debug!(
            "Doing basis check against num arcs {}",
            self.agent_arqs.len()
        );
```

Reply:
Probably a bit specific to this issue anyway, I'm happy to just drop these